### DISCUSSION TITLE
Welcome to stars-nova Discussions!

### DISCUSSION THREAD

**Discussion by ekolis** [ekolis/stars-nova#61] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mooskagh** [comment]
Does the mechanics of this game exactly copies the one from "Stars!"?
If yes, why is it? Was Stars! reverse engineered, or was just so well documented so that it's easy to replicate?

**Reply by ekolis** [reply]
Yes, it does attempt to exactly copy Stars! Honestly, I'm not sure how. I'm leaning toward "well-documented". I wasn't the original creator of this project; I just made this repo because the original creator seemed to have abandoned his repo on SourceForge. But now someone's picked that up again and I haven't gotten much done here, so maybe I should see if I can help out there...
